### ISSUE TITLE
Fix parseErrors for single-line comments in selector-*

### ISSUE THREAD

**Issue by SevenOutman** [stylelint/stylelint#5257]
> Clearly describe the bug

When running against single-line comment between multi-line selectors, stylelint crashes with the following message

    Unexpected '/'. Escaping special characters with \ may help.

> Which rule, if any, is the bug related to?

None.

> What code is needed to reproduce the bug?

```less
a,
// crash
b {
  width: 00.1em;
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard"
}
```

> Which version of stylelint are you using?

`13.12.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "src/**/*.less" --syntax less`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to [Less inline comments](https://lesscss.org/#comments).

> What did you expect to happen?

Don't crash.

> What actually happened (e.g. what warnings or errors did you get)?

Stylelint crashes with the following message

![image](https://user-images.githubusercontent.com/8225666/115525836-0cfa0880-a2c2-11eb-8c8e-f1f049f9f47e.png)

**Comment by jeddy3** [comment]
@SevenOutman Thanks for the report and for using the template.

I'm unable to replicate a crash locally. Instead, `parseErrors` are caught and rendered by the formatter.

I'm not sure what we can do about the `parseErrors` as the selector parser we use is designed for CSS.

I'll label as "ready to implement" in case anyone has an idea of how we could address this.
